docs/user/faq.rst

```diff
@@ -668,17 +668,37 @@ The `stream` of a body part is a file-like object implementing the ``read()``
 method, making it compatible with ``boto3``\'s
 `upload_fileobj <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.upload_fileobj>`_:
```

Comment:
we could maybe add also `aioboto3` link? or at least mention that asgi uses that library to wrap the boto3 config

Reply:
Right, good idea 👍 I'll add a note.